### DISCUSSION TITLE
Permission Denied error

### DISCUSSION THREAD

**Discussion by AashiDutt** [GoogleCloudPlatform/generative-ai#98] [General]
Hi, 

I tried running the intro PaLM API notebook in collab and provided all authorizations, and project id, and enabled the vertex API. However, I got an error while running the first "Hello PaLM prompt". 
 It returned an error - Permission denied.

Please suggest how to proceed further.

**Comment by polong-lin** [comment]
Hi @AashiDutt , when using Colab, you might need to first authenticate your Colab instance with Google Cloud, by running:
```
from google.colab import auth
auth.authenticate_user()
```

Then try to run:
```
PROJECT_ID = "your-project-id"
LOCATION = "" #e.g. us-central1

import vertexai
vertexai.init(project=PROJECT_ID, location=LOCATION)
```

Could you let us know if you are still running into any authentication issues?

**Reply by AashiDutt** [reply]
Hello @polong-lin 

Thank you for reaching out. This issue has been resolved, seemed like some billing problem to enable vertex ai on the console. 

Thanks
